index.html

```diff
@@ -0,0 +1,152 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Tangerine">
+    <title>MEdicore</title>
+    <link href="./styles.css" rel="stylesheet">
+</head>
+<body>
+    <div class="section--pink">
```

Comment:
## Naming Modifier is not correct
I think this way is not good way to add the modifier `.section--pink`. CSS class is formed as block’s or element’s name plus two dashes then you add the name of the modifier.  you can follow this link [here BEM-Naming](https://getbem.com/naming/)

Reply:
Thanks i didn't notice